torch/_functorch/pytree_hacks.py

```diff
@@ -4,21 +4,19 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-from torch.utils._pytree import tree_flatten, tree_unflatten
+import warnings
 
+# TODO: remove this file when the migration of the pytree utility is done
+from torch.utils._pytree import tree_map_, treespec_pprint
 
-def tree_map_(fn_, pytree):
-    flat_args, _ = tree_flatten(pytree)
-    [fn_(arg) for arg in flat_args]
-    return pytree
 
+__all__ = ["tree_map_", "treespec_pprint"]
 
-class PlaceHolder:
-    def __repr__(self):
-        return '*'
```

Comment:
Was this unused in the codebase?

Reply:
Yes. It is only referenced by `treespec_pprint`.